lib/neo4j/migrations/schema_migration.rb

```diff
@@ -0,0 +1,11 @@
+module Neo4j
+  module Migrations
+    class SchemaMigration
+      include Neo4j::ActiveNode
+
+      property :migration_id, type: String, constraint: :unique
```

Comment:
Perhaps this could be the `id_property` because it's a natural identifier and then we don't need to worry about `uuid` (which would just mean more data returned from the DB anyway)